flink-state-backends/flink-statebackend-changelog/src/main/java/org/apache/flink/state/changelog/restore/ChangelogBackendLogApplier.java

```diff
@@ -107,16 +112,36 @@ private static void applyMetaDataChange(
         }
     }
 
+    private static Optional<StateTtlConfig> readTtlConfig(DataInputView in) throws IOException {
```

Comment:
Since `StateTtlConfig` has the meaning of enabled or disabled by itself, why we need to introduce an `Optional` here? If you want to reduce the writing content during serialization, you can just return a `StateTtlConfig.DISABLED` here if read false.